docker/deployment/db-deployment.yaml

```diff
@@ -39,6 +39,12 @@ spec:
   selector:
     matchLabels:
       app: db
+  strategy:
+      # we don't ever want more than 1 "db" running at any time since the pvc above is RWO
+      type: RollingUpdate
+      rollingUpdate:
+        maxSurge: 0
+        maxUnavailable: 25%
```

Comment:
These can be removed assuming that `type: Recreate` is what we want

Reply:
agreed, I would prefer this simpler config as well.